ui/src/common/components/KeywordList.jsx

```diff
@@ -4,10 +4,15 @@ import { List } from 'immutable';
 
 import ExpandableInlineList from './ExpandableInlineList';
 import UnclickableTag from './UnclickableTag';
+import Latex from './Latex';
 
 function renderKeyword(keyword) {
   const keywordValue = keyword.get('value');
-  return <UnclickableTag color="blue">{keywordValue}</UnclickableTag>;
+  return (
+    <UnclickableTag color="blue">
+      <Latex>{keywordValue}</Latex>
```

Comment:
Adding this here will also have an effect on literature, seminars and conferences too since they are using this component. Is that something we  are ok with ? cc @drjova